qa/L0_lifecycle/test.sh

```diff
@@ -227,8 +227,8 @@ for i in graphdef netdef plan ; do
 done
 cp -r $DATADIR/qa_model_repository/savedmodel_float32_float32_float32 .
 
-SERVER_ARGS="--model-store=`pwd`/models --allow-poll-model-repository=false \
-             --repository-poll-secs=1 --exit-timeout-secs=5"
+SERVER_ARGS="--model-store=`pwd`/models --allow-poll-model-repository=true \
```

Comment:
This is confusing because --allow seems to turn it on but --repository-poll-secs=0 turns if off. Can you just use --allow-poll-model-repository=false and remove --repository-poll-secs?

Reply:
This is how we can tell the server to only poll model repository once. We should also update this usage then (i.e. whether loading the model repository on startup depends on `allow-model-control`)
> --repository-poll-secs	Interval in seconds between each poll of the model repository to check for changes. A value of zero indicates that the repository is checked only a single time at startup. Valid only when --allow-poll-model-repository=true is specified.

Reply:
Oh, that makes sense.... But --allow-poll-model-repository=false is redundant with true and poll-secs=0... but that is just the way it is.
Yes we need to add the allow-model-control flag and update the descriptions. But default values should be set so current behavior is maintained